src/HeuristicCompletion-Tests/CoCompletionEngineTest.class.st

```diff
@@ -28,6 +31,19 @@ CoCompletionEngineTest >> newCompletionEngine [
 	^ CoCompletionEngine new
 ]
 
+{ #category : 'accessing' }
+CoCompletionEngineTest >> setUp [
+	super setUp.
+	oldSort := CompletionContext sorterClass.
+	CompletionContext sorterClass: NoSorter.
+]
+
+{ #category : 'accessing' }
+CoCompletionEngineTest >> tearDown [
+	super tearDown.
+   CompletionContext sorterClass: oldSort.
```

Comment:
Please tab!

Reply:
it's already tabed in my image ... , i don't know why it's like this here.
![Uploading Capture d’écran 2024-12-01 à 18.32.52.png…]()

Reply:
@omarabedelkader Because you used 4 spaces instead of 1 tab

Reply:
right !
i think now it's fixed.